From d894c678a9f916bf10590245a0dd108b8af98ed1 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Tue, 20 May 2014 14:07:03 +0800
Subject: [PATCH 2/3] iw/partition_gui.py: only support standard partition

We only support the standard partitions in this delivery, so disable the
lvm and dmraid partitions.

Upstream-Status: Inappropriate [WR specific]

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 pyanaconda/iw/partition_gui.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/pyanaconda/iw/partition_gui.py b/pyanaconda/iw/partition_gui.py
index c62d0a6..ce3e5d9 100644
--- a/pyanaconda/iw/partition_gui.py
+++ b/pyanaconda/iw/partition_gui.py
@@ -1341,6 +1341,11 @@ class PartitionWindow(InstallWindow):
                         custom_icon="warning")
             return
 
+        # Only support standard partitions atm.
+        tempformat = self.storage.defaultFSType
+        device = self.storage.newPartition(fmt_type=tempformat)
+        self.editPartition(device, isNew = True)
+        return
         # We will activate the create lv button when we have a VG to put the
         # LVs on.
         activate_create_lv = False
-- 
1.8.3.1

